doc/languages-frameworks/go.section.md

```diff
@@ -0,0 +1,156 @@
+# Go
```

Comment:
Anchors should be preserved to avoid breaking links:

```suggestion
# Go {#sec-language-go}
```

Reply:
done.